Without this change pgrep seems to return the PID of the abduco process
spawned in the previous line because it may not be removed yet. Removing
the pgrep invocation is not the best solution but good enough for now.

diff -upr abduco-0.6.orig/testsuite.sh abduco-0.6/testsuite.sh
--- abduco-0.6.orig/testsuite.sh	2019-02-14 11:33:19.521290879 +0100
+++ abduco-0.6/testsuite.sh	2019-02-14 11:33:29.361298623 +0100
@@ -55,7 +55,6 @@ expected_abduco_detached_output() {
 
 check_environment() {
 	[ "`$ABDUCO | wc -l`" -gt 1 ] && echo Abduco session exists && exit 1;
-	pgrep abduco && echo Abduco process exists && exit 1;
 	return 0;
 }
 
